### ISSUE TITLE
New post notification: Template in site language, not receiving user's language

### ISSUE THREAD

**Issue by allilevine** [Automattic/wp-calypso#95725]
### Quick summary

New post notifications for sites in non-EN languages are sent with the template translated into the site locale, instead of the user's language.

![Image](https://github.com/user-attachments/assets/1fa549fe-0175-494e-8b1b-2b664f598774)

### Steps to reproduce

1. Subscribe to a non-EN site.
2. Get new post notifications via email.

### What you expected to happen

The email template to be in my user language.

### What actually happened

The email template is in the site language.

### Impact

Some (< 50%)

### Available workarounds?

No but the platform is still usable

### Platform (Simple and/or Atomic)

Simple

### Logs or notes

Sent via the WordPress.com HTML email class: fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Szh%2Qcyhtvaf%2Sugzy%2Qrznvyf%2Sugzy%2Qrznvy%2Qpynff.cuc%3Se%3Q2o735rq0-og

Reported by @kriskarkoski here: p1729864651397559-slack-C03NLNTPZ2T

**Comment by simison** [comment]
Hi! This works exactly as intended. See D151251-code and peKye1-Sn-p2

Otherwise, we'd have a mixture of languages in the same email; one of site content and different for the UI, which would be confusing to read even if you happen to understand both languages.

Please feel free to ping Zap about any issues with emails in future. Thanks!

**Comment by allilevine** [comment]
> Please feel free to ping Zap about any issues with emails in future. Thanks!

Thank you, good to know! 

@kriskarkoski Does this resolve the issue you were seeing?

**Comment by kriskarkoski** [comment]
Not really, I would expect things about my account (like unsubscribing) to be in my language, not the user's, which I have no control over. Maybe I'm translating someone's post content, I'd still want information for me to be something I can read by default.

**Comment by simison** [comment]
Thanks! Yeah, those are perfectly logical reasonings. I don't have links now, but we've had gotten customer feedback about mixed languages being a problem.

We consider the entire email template to "be from the publisher", even if it has some "platform" UIs. Similarly, we wouldn't show "like/share/reblog" buttons or the comment form in user's language at the site, even if those are coming from "platform" and also include bits of UIs for authentication and managing your profile or subscriptions.

I'll use a full-email template screenshot as I think it better illustrates that there's more outside footer's "manage" link in the emails that publisher currently cannot modify. I'll also use more recent template with updated footer that we're working on shipping soon. In red, the content comes from a user, and the rest is from the email template.

![Image](https://github.com/user-attachments/assets/3b891fd0-2f90-4ecf-919e-3bbae2a01e46)

Here it's hopefully clearer how seeing a mixture of languages changing section by section would be confusing and hard to parse:
- 
- Header CTAs
- Title
- Author & date
- Content
- CTAs
- Footer

It becomes especially restless when those languages are a mixture of left-to-right/right-to-left or latin vs other scripts. Cognitively it's hard to read text which changes language constantly.

We will in future give publishers more control over the rest of content, like the ability to hide our branding and modify the address line;  the template becomes more "theirs" and allows them to remove "us" from the email.

Finally, readers likely are not subscribing to blogs written in languages they don't understand, so there shouldn't be issues with understanding the email fully.
